packages/zoe/src/contractSupport/zoeHelpers.js

```diff
@@ -123,10 +123,12 @@ export const fitProposalShape = (seat, proposalShape) =>
  * null contents. If the client submits an offer which does not match
  * these expectations, the seat will be exited (and payments refunded).
  *
+ * @deprecated Use optional `proposalShape` argument to `makeInvitation` with
+ * a genuine pattern.
  * @param {ZCFSeat} seat
  * @param {ExpectedRecord} expected
  */
-export const assertProposalShape = (seat, expected) => {
+export const assertProposalShapeDeprecated = (seat, expected) => {
```

Comment:
imo the `@deprecated` annotation is sufficient since TS-aware IDE will render `assertProposalShape` in a strikeout font. But not opposed to this renaming.

Reply:
Done